nominatim/tools/country_info.py

```diff
@@ -72,15 +83,24 @@ def setup_country_tables(dsn, sql_dir, ignore_partitions=False):
             else:
                 partition = props.get('partition')
             lang = props['languages'][0] if len(props['languages']) == 1 else None
-            params.append((ccode, partition, lang))
-
+            name = add_prefix_to_keys(props.get('names')
+                 .get(_COUNTRY_INFO.key_prefix()), _COUNTRY_INFO.key_prefix())
+            name = json.dumps(name, ensure_ascii=False, separators=(', ', '=>'))
+            params.append((ccode, name[1:-1], lang, partition))
```

Comment:
You shouldn't need the conversion to json for name anymore here. psycopg should do the conversion for you. If it doesn't, then you need the [hstore adapter](https://www.psycopg.org/docs/extras.html?highlight=hstore#psycopg2.extras.register_hstore). Add a line `psycopg2.extras.register_hstore(cur)` below before the `execute()`.